CMakeLists.txt

```
@@ -112,6 +112,10 @@ add_subdirectory(sdk/keyvault)
 add_subdirectory(sdk/storage)
 add_subdirectory(sdk/template)
 
+if(ENABLE_AZURE_CORE_AMQP)
+#messaging sdks need AMQP
+add_subdirectory(sdk/eventhubs)
```

same nit: indentation
```suggestion
  #messaging sdks need AMQP
  add_subdirectory(sdk/eventhubs)
```